arduino-cli/.gitpod.yml

```diff
@@ -1,4 +1,3 @@
 tasks:
-- command: go get -v
+- command: cd arduino-cli && go get -v
 checkoutLocation: arduino-cli
```

Comment:
Is this necessary?

Reply:
yes as the tasks get started in the `workspaceLocation` which is `/workspace`